docs/development/pigs/pig-026.rst

```diff
@@ -0,0 +1,91 @@
+.. include:: ../../references.txt
+
+.. _pig-026:
+
+********************************
+PIG 26 - Priors and FitStatistic
+********************************
+
+* Author: Noah Biederbeck
+* Created: ``2023-03-13``
+* Accepted:
+* Status: draft
+* Discussion: `#4381`_
+
+Abstract
+========
+
+This PIG is intended to introduce priors on models that are evaluated during fitting.
+
+Motivation
+==========
+
+Using priors on models and on parameters is the next step in full-fledged statistical analyses.
+A prior can incorporate the analysers' knowledge of the topic at hand, and yield more realistically and trustworthy results.
```

Comment:
Maybe not only the analyser... Indeed, one idea to deal with the IRFs systematics was to use the priors mechanics. So, if it is implemented like that, the knowledge comes from the people making the IRFs...

By the use, do we add as use case the inclusion of IRFs systematics into priors?